src/preact/base-element.js

```diff
@@ -411,13 +544,17 @@ export class PreactBaseElement extends BaseElement {
    * This has no effect on Bento documents, since they lack an Actions system.
    * Instead, they should use `(await element.getApi()).action()`
    * @param {string} alias
-   * @param {function(!API_TYPE, !../service/action-impl.ActionInvocation)} handler
-   * @param {../action-constants.ActionTrust_Enum} minTrust
+   * @param {function(API_TYPE, *):void} handler
+   * @param {ActionTrust_Enum} minTrust
    * @protected
    */
   registerApiAction(alias, handler, minTrust = ActionTrust_Enum.DEFAULT) {
     this.registerAction?.(
       alias,
+      /**
+       * @param {*} invocation
+       * @return {void}
```

Comment:
Is a return type annotation necessary for a void return?

Reply:
removed!